react/class_components/component_lifecycle_methods.md

```diff
@@ -0,0 +1,55 @@
+### Introduction
+
+In the previous section, you've dealt with the component lifecycle through the useEffect function. This is how you've run code based on a components lifecycle stage so far, but in class components we have to use specific lifecycle methods instead.
```

Comment:
Wording suggestion:

```suggestion
In the previous section, you've learned about a component's lifecycle. In functional components, we've primarily used the `useEffect` hook to take perform tasks throughout the lifecycle, but in class components we have to use specific lifecycle methods instead.
```

Reply:
Looks good to me, is the take before perform a typo? Can't make sense of it in my head.